cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -86,6 +87,11 @@ object ReporterCommand : CommandWithHelp() {
             order = PARAMETER_ORDER_OPTIONAL)
     private var postProcessingScript: File? = null
 
+    @Parameter(description = "A file containing a blacklist of copyright statements.",
+            names = ["--copyright-blacklist-file"],
```

Comment:
I don't think "blacklist" is a good term here, as in the larger context of license governance it has a notion of "not allowing the use of blacklisted licenses" (even if this is about copyright statements only here). We should make more clear from the name that the goal / purpose is not to filter out blacklisted copyright holders, but to remove garbage / lines of text that simply are no copyright statements.

So how about frankly calling this `--copyright-garbage`?